pandas/tests/scalar/timestamp/test_comparisons.py

```diff
@@ -324,5 +324,4 @@ def __eq__(self, other) -> bool:
     for left, right in [(inf, timestamp), (timestamp, inf)]:
         assert left > right or left < right
         assert left >= right or left <= right
-        assert not (left == right)
```

Comment:
Probably should keep since this ensures both `__eq__` and `__ne__` are being tested

Reply:
Makes sense. Thanks for the insight.